src/sentry/static/sentry/app/components/organizations/headerItem.tsx

```diff
@@ -20,7 +20,10 @@ type Props = {
   locked: boolean;
   loading: boolean;
   onClear: () => void;
-} & React.HTMLProps<HTMLDivElement>;
+  children: React.ReactNode;
+  className: string;
+  forwardRef?: React.Ref<HTMLDivElement>;
+} & Pick<React.HTMLProps<HTMLDivElement>, 'onClick'>;
```

Comment:
I would typically just write this as `onclick: React.HTMLProps<HTMLDivElement>['onClick']` in the props list.

Personal preference maybe tho.

Reply:
Yeah I think it's just a pref -- with `Pick<>` we could add more props easily.